pages/item/_itemId/index.vue

```diff
@@ -380,21 +304,27 @@
 <script lang="ts">
 import Vue from 'vue';
 import { mapActions } from 'vuex';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment -- Temporary module while waiting for fixes to language names on the server
-// @ts-ignore
-import langs from 'langs';
 import {
   BaseItemDto,
   BaseItemPerson,
   ImageType,
-  MediaSourceInfo,
-  MediaStream
+  MediaSourceInfo
 } from '@jellyfin/client-axios';
 import imageHelper from '~/mixins/imageHelper';
 import formsHelper from '~/mixins/formsHelper';
 import itemHelper from '~/mixins/itemHelper';
+import TrackSelector from '~/components/Item/TrackSelector.vue';
+
+interface TwoColsInfoColumn {
+  lCols: number;
+  lSm: number;
+  rCols: number;
+  rSm: number;
+  lClass: { [key: string]: boolean };
+}
 
 export default Vue.extend({
+  components: { TrackSelector },
```

Comment:
This is not needed as we have all the components autoimported

Reply:
Done